manager/controllers/default/resource/create.class.php

```diff
@@ -249,14 +249,20 @@ public function getDefaultTemplate() {
             'FCSet.active' => true,
             'Profile.active' => true,
         ));
+
+        $criteriaUserGroups = [];
+
+        if(!empty($userGroups)){
+            $criteriaUserGroups['ProfileUserGroup.usergroup:IN'] = $userGroups;
+        }
+
+        $criteriaUserGroups[] = array(
+            'OR:ProfileUserGroup.usergroup:IS' => null,
```

Comment:
Should this OR be there if the previous condition isn't added?

Reply:
As far I know this OR will be ignored by xpdo when the previous condition isn't added.

When logging the statement in MODX the following occurs:

Without the $userGroups

> AND  (  ( `ProfileUserGroup`.`usergroup` IS NULL AND `UGProfile`.`active` = 1 )  OR `ProfileUserGroup`.`usergroup` IS NULL )  )

With $userGroups:

> AND  (  ( `ProfileUserGroup`.`usergroup` IN ('1') OR  ( `ProfileUserGroup`.`usergroup` IS NULL AND `UGProfile`.`active` = 1 )  )  OR `ProfileUserGroup`.`usergroup` IS NULL )  )

side note which has nothing to do with this  bug.

> OR ( ProfileUserGroup.usergroup IS NULL AND UGProfile.active = 1 ) 

This nested OR seems to be redundant.